apps/condo/pages/settings/index.tsx

```diff
@@ -68,48 +68,48 @@ const SettingsPage: React.FC = () => {
         return result
     }, [hasSubscriptionFeature, canManageContactRoles])
 
-    const settingsTabs: SettingsTabPaneDescriptor[] = useMemo(
+    const settingsTabs: TabItem[] = useMemo(
         () => [
             hasSubscriptionFeature && {
                 key: SETTINGS_TAB_SUBSCRIPTION,
-                title: SubscriptionTitle,
-                content: <SubscriptionPane/>,
+                label: SubscriptionTitle,
+                children: <SubscriptionPane/>,
             },
             {
                 key: SETTINGS_TAB_PROPERTY_HINT,
-                title: HintTitle,
-                content: <TicketPropertyHintSettings/>,
+                label: HintTitle,
+                children: <TicketPropertyHintSettings/>,
             },
             {
                 key: SETTINGS_TAB_PROPERTY_SCOPE,
-                title: PropertyScopeTitle,
-                content: <PropertyScopeSettingsContent/>,
+                label: PropertyScopeTitle,
+                children: <PropertyScopeSettingsContent/>,
                 eventName: 'PropertyScopeVisitIndex',
             },
             {
                 key: SETTINGS_TAB_EMPLOYEE_ROLES,
-                title: EmployeeRolesTitle,
-                content: <EmployeeRolesSettingsContent/>,
+                label: EmployeeRolesTitle,
+                children: <EmployeeRolesSettingsContent/>,
             },
             {
                 key: SETTINGS_TAB_PAYMENT_DETAILS,
-                title: DetailsTitle,
-                content: <RecipientSettingsContent/>,
+                label: DetailsTitle,
+                children: <RecipientSettingsContent/>,
             },
             canManageContactRoles && {
                 key: SETTINGS_TAB_CONTACT_ROLES,
-                title: RolesTitle,
-                content: <ContactRolesSettingsContent/>,
+                label: RolesTitle,
+                children: <ContactRolesSettingsContent/>,
             },
             {
                 key: SETTINGS_TAB_CONTROL_ROOM,
-                title: ControlRoomTitle,
-                content: <ControlRoomSettingsContent/>,
+                label: ControlRoomTitle,
+                children: <ControlRoomSettingsContent/>,
             },
             canManageMobileFeatureConfigsRoles && hasMobileFeatureConfigurationFeature && {
                 key: SETTINGS_TAB_MOBILE_FEATURE_CONFIG,
-                title: MobileFeatureConfigTitle,
-                content: <MobileFeatureConfigContent/>,
+                label: MobileFeatureConfigTitle,
+                children: <MobileFeatureConfigContent/>,
             },
         ].filter(Boolean),
         [hasSubscriptionFeature, hasMobileFeatureConfigurationFeature, SubscriptionTitle, HintTitle, PropertyScopeTitle, EmployeeRolesTitle, DetailsTitle, canManageContactRoles, RolesTitle, ControlRoomTitle, MobileFeatureConfigTitle],
```

Comment:
Check deps array, when I was checking it at morning before this task went to you, there was a warning (for mobile config) and in `tabKeysToDisplay` too, can you fix it here?

(And are your eslint is turned on? 🙄)